docs/advanced-guides/posts.md

```diff
@@ -364,16 +364,28 @@ The blob object, as JSON, would look something like:
 }
 ```
 
-The blob is then included in a `app.bsky.embed.images` array, along with an alt-text string. The `alt` field is required for each image. Pass an empty string if there is no alt text available.
+The blob is then included in a `app.bsky.embed.images` array, along with an alt-text string and the image dimensions.
+
+The `alt` field is required for each image. Pass an empty string if there is no alt text available.
```

Comment:
This isn't true at the API / Lexicon level, right? I think it is an optional field? I think better to not have it than pass an empty string. I would just remove this added line for now.

Reply:
this is an old line fwiw, but happy to remove